apps/mobile/src/features/send/send-sheet-content.tsx

```diff
@@ -0,0 +1,21 @@
+import { useSendContext } from './send-context';
+import { SelectAccount } from './send-steps/select-account';
+import { SelectAsset } from './send-steps/select-asset';
+import { SendForm } from './send-steps/send-form';
+
+export function SendSheetContent() {
+  const { step } = useSendContext();
+
+  switch (step) {
+    case 'select-account':
+      return <SelectAccount />;
+    case 'select-asset':
+      return <SelectAsset />;
+    case 'send-form':
+      return <SendForm />;
+    case 'approver':
+      return <></>;
```

Comment:
You can skip this `return <></>;` and have:

```
case 'approver':
default:
      return <></>;

```

Reply:
Not too sure if I'll need this, but if I do there will be something diff from the default for the approver content. Sorry, I prob should have left comments that some of this is just setup.

Reply:
No problem 👍 just mentioned some stuff as I was going through it. 

PR good to go for me as is